web/src/app/admin/AdminNumberLookup.tsx

```diff
@@ -50,24 +50,27 @@ const numInfoQuery = gql`
 export default function AdminNumberLookup(): JSX.Element {
   const [number, setNumber] = useState('')
   const [staleCarrier, setStaleCarrier] = useState(true)
-  const [lastError, setLastError] = useState(null as null | ApolloError)
 
-  const { data: numData } = useQuery(numInfoQuery, {
+  const [{ data: numData, error: queryError }] = useQuery({
+    query: numInfoQuery,
     variables: { number },
-    pollInterval: 0,
-    onError: (err) => setLastError(err),
   })
   const numInfo = numData?.phoneNumberInfo as PhoneNumberInfo
 
-  const [lookup, { data: carrData, loading: carrLoading }] = useMutation(
-    carrierInfoMut,
-    {
-      variables: { number },
-      onError: (err) => setLastError(err),
-    },
-  )
+  const [
+    { data: carrData, fetching: carrLoading, error: mutationError },
+    commit,
+  ] = useMutation(carrierInfoMut)
   const carrInfo = carrData?.debugCarrierInfo as DebugCarrierInfo
 
+  const [lastError, setLastError] = useState<CombinedError>()
+  useEffect(() => {
+    setLastError(queryError)
+  }, [queryError])
+  useEffect(() => {
+    setLastError(mutationError)
+  }, [mutationError])
```

Comment:
We need a conditional on each to only set the error if there is one.

i.e. `if (queryError) setLastError(queryError)`